client/src/components/SignUp.style.js

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
+  background-color: #f9d52f;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const Title = styled.h1`
+  fontfamily: Work Sans;
```

Comment:
these styles are not being read as they should have hyphens! but don't worry as these styles are being read in the global styles